src/patternfly/layouts/Page/page.scss

```diff
@@ -135,6 +136,13 @@
   .pf-l-page__header-tools {
     padding-top: var(--pf-l-page__header-tools--MarginTop);
     padding-bottom: var(--pf-l-page__header-tools--MarginBottom);
+
+    .pf-c-dropdown__toggle-text {
+      max-width: var(--pf-l-page__header-tools-c-dropdown__toggle-text--MaxWidth);
+      overflow: hidden;
+      text-overflow: ellipsis;
+      white-space: nowrap;
+    }
```

Comment:
do you think this is a better approach than absolutely positioning the icon? You could save having to create this element if the icon was absolutely positioned.

Reply:
maybe I'm thinking of this wrong but would that be flexible enough? What if there was a border on the dropdown?